### PR TITLE
web-wallet: Fix build warning for unused icon

### PR DESCRIPTION
**Pull request by nortonandreev**
Resolves #1632

### CODE REVIEW

File: web-wallet/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Fix build warning for unused icon in toast within on:click handler [#1632]
```
Comment (ascartabelli) [inline web-wallet/CHANGELOG.md:12]: This change shouldn't be in the changelog.

File: web-wallet/src/routes/components-showcase/Toasts.svelte [hunk 2]
```diff
@@ -2,26 +2,25 @@
...
     size="small"
-    on:click={() => toast("success", "Toast message", mdiCopyleft)}
+    on:click={triggerToast}
   />
 
   <Button
     text="toggle another toast"
     variant="secondary"
     size="small"
-    on:click={() =>
```
Comment (ascartabelli) [inline web-wallet/src/routes/components-showcase/Toasts.svelte:19]: This way you're removing a test case (two actually: error + text wrapping).

**Review by ascartabelli — CHANGES_REQUESTED** [review]
Left a couple of notes.
This effectively solves the problem, thanks, as it was annoying.
I think we need to understand better why the build system sees a problem here.

One last thought: should we remove the `components-showcase` from the production build altogether?

File: web-wallet/src/routes/components-showcase/Toasts.svelte (hunk 2 above)
Comment (nortonandreev) [reply web-wallet/src/routes/components-showcase/Toasts.svelte:19]: I didn't even see it, have added it back.

Generally, I wouldn't expect the components showcase to be used to cover different test scenarios, although I agree it's useful to have more variations of a component.

Comment (ascartabelli) [reply web-wallet/src/routes/components-showcase/Toasts.svelte:19]: Well, it doesn't replace a storybook, obviously, but it has been useful to quickly check the styles: so variations are welcome.

**Review by ascartabelli — APPROVED** [review]

**Pull request merged**